<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncapsulateField::GlobalConfigPage</class>
 <widget class="QWidget" name="EncapsulateField::GlobalConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_patterns">
     <property name="title">
      <string>Patterns</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="patterns"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="add">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset theme="list-add">
              <normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset theme="edit-delete">
              <normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveUp">
            <property name="text">
             <string>Move Up</string>
            </property>
            <property name="icon">
             <iconset theme="arrow-up">
              <normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveDown">
            <property name="text">
             <string>Move Down</string>
            </property>
            <property name="icon">
             <iconset theme="arrow-down">
              <normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</normaloff>../../../../../../tmp/kdevelop/plugins/projectfilter</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
